Helper/GraphQL/Client.php

```diff
@@ -194,4 +194,21 @@ public function getFeatureSwitches() {
 
         return $res;
     }
+
+    /**
+     * This method sends the logs passed in to Bolt.
+     *
+     * @param $jsonEncodedLogArray
+     * @throws LocalizedException
+     */
+    public function sendLogs($jsonEncodedLogArray) {
```

Comment:
Can you add a unit test? I imagine it'll look very similar to `testGetFeatureSwitches_success`

Reply:
omg I saw the tests but my brain didn't register them.